packages/lb-components/src/components/subAttributeList/index.tsx

```diff
@@ -37,24 +55,41 @@ const SubAttributeList = (props: IProps) => {
                 {t('SubAttribute')}-{subAttribute.key}
               </div>
               {subAttribute.subSelected?.length < 5 ? (
-                <AttributeList
-                  list={subAttribute.subSelected.map((v: IInputList) => ({
-                    label: v.key,
-                    value: v.value,
-                  }))}
-                  selectedAttribute={getValue(subAttribute)}
-                  num='-'
-                  forbidColor={true}
-                  forbidDefault={true}
-                  attributeChanged={(value) => setSubAttribute(subAttribute.value, value)}
-                  style={{ marginBottom: 12 }}
-                />
+                subAttribute?.isMulti ? (
+                  <Checkbox.Group
```

Comment:
why not implement it into AttributeList

Reply:
- Functions that do not require multiple choices
- Adding unnecessary test coverage